playground/src/components/contract/contract.tsx

```diff
@@ -430,7 +428,8 @@ export function ContractComponent() {
               </FormGroup>
             </div>
           </div>
-          {contractArtifact.functions
+          {contractArtifact.nonDispatchPublicFunctions
+            .concat(contractArtifact?.functions ?? [])
```

Comment:
These are different types - is that ok for this fn?

Reply:
Yup, since one extends the other and I only access the stuff on the Abi